credentials/sources.go

```diff
@@ -99,11 +99,23 @@ func getCredentialsFromBytes(byteArray []byte) ([]Credentials, error) {
 	)
 
 	methods := []func(bytes []byte) ([]map[string]interface{}, error){tryReadingList, tryReadingMapOfCredentials, tryReadingSingleCredential}
+	var success = false
```

Comment:
The most go-diomatic way to init with a value is the walrus `:=` operator like so 
```suggestion
	success := false
```